pkg/scheduler/framework/types.go

```diff
@@ -234,7 +234,11 @@ func (f *FitError) Error() string {
 	reasons := make(map[string]int)
 	for _, status := range f.Diagnosis.NodeToStatusMap {
 		for _, reason := range status.Reasons() {
-			reasons[reason]++
+			if len(status.failedPlugin) > 0 && klog.V(8).Enabled() {
+				reasons[fmt.Sprintf("%v failed plugins: %v", reason, status.failedPlugin)]++
```

Comment:
```suggestion
				reasons[fmt.Sprintf("%v, failed by %v", reason, status.failedPlugin)]++
```

And it worths a release note, to indicate that when V(X) is enabled, the scheduling failure message contains a "failed by <plugin>" info.

Reply:
thanks for the suggestion, have updated to add a log entry instead.